src/utilities/extendSchema.js

```diff
@@ -379,16 +399,18 @@ export function extendSchema(
     });
   }
 
-  function extendArgsMap(
-    args: GraphQLArgument[],
-  ): GraphQLArgument[] {
-    return args.map(arg => ({
-      name: arg.name,
-      type: extendFieldType(arg.type),
-      defaultValue: arg.defaultValue,
-      description: arg.description,
-      astNode: arg.astNode,
-    }));
+  function extendArgs(args: GraphQLArgument[]): GraphQLFieldConfigArgumentMap {
+    return keyValMap(
+      args,
+      arg => arg.name,
+      arg => ({
+        name: arg.name,
+        type: extendFieldType(arg.type),
```

Comment:
Not a critical issue but you could think of a better name for this function since now it also used for arguments. How about `extendType`?

Reply:
`extendType` is already used in this scope. Should I move the logic of `extendFieldType` into this function or used something like `extendValueType` ?

Reply:
@mgrenier I think it should be `extendType` and `extendNamedType` but it will make this review hard to read so I created #1327.